app/javascript/pages/admin/IndividualEvents/index.js

```diff
@@ -112,6 +112,7 @@ class IndividualEvents extends React.Component {
           </select>
           <button
             className={s.bulkActionButton}
+            disabled={!this.state.bulkAction}
```

Comment:
I'm not sure about the use of `state` here. Ideally state management should be centralized in Apollo. That said, state management is out of scope of this PR and the PR itself LGTM.

Reply:
It's just the component state, it isn't shared by anything outside this component so it should be fine I think. Could be a candidate for react hooks :)